src/transformer/events/mod_assign/assignment_submitted.php

```diff
@@ -34,7 +36,7 @@ function assignment_submitted(array $config, \stdClass $event) {
                 $lang => 'submitted'
             ],
         ],
-        'object' => utils\get_activity\event_module($config, $event, $lang),
+        'object' => utils\get_activity\course_assignment($config, $event->contextinstanceid, $assignment->name, $lang),
         'timestamp' => utils\get_event_timestamp($event),
         'context' => [
```

Comment:
The displayed xapi statement has erroneous data in context.extensions -
[completedAssignment.txt](https://github.com/xAPI-vle/moodle-logstore_xapi/files/2249025/completedAssignment.txt)

Reply:
What is the erroneous data?

Reply:
I can't see what's erroneous in the context extensions there

Reply:
Line 35-36 ?

Reply:
Line 35-36 in the attached statement, I mean

Reply:
This bit here?
```
"event_name": "\\core\\event\\course_module_completion_updated",
"event_function": "\\src\\transformer\\events\\core\\course_module_completion_updated"
```

That looks like a different event, is it possible that the `\mod_assign\event\assessable_submitted` just isn't being sent, but it also triggers the completion event in the background?

Reply:
Or is 

"event_name": "\\core\\event\\course_module_completion_updated",
"event_function": "\\src\\transformer\\events\\core\\course_module_completion_updated"

intentional?

Reply:
This seems ok now

Reply:
Definitely not intentional for `\mod_assign\event\assessable_submitted`

Reply:
You're right - I was thinking \core\event\course_module_completion_updated meant something else. That means that \mod_assign\event\assessable_submitted is not being sent to the lrs.

Reply:
Okay we'll figure out this one tomorrow.